bin/docserv-stitch

```diff
@@ -202,6 +231,7 @@ for file in *.xml; do
     result=$( \
       xmllint="$xmllint" jing="$jing" starlet="$starlet" \
       valid_languages="$valid_languages_sorted" \
+      site_sections="$site_sections" \
```

Comment:
I think renaming this variable to `valid_site_sections` may help clarify intent, especially, since you're checking against this in your later check script.

Reply:
Good idea! That's consistent with the language part. Will do.